### PR TITLE
Update quick start page

### PR DESCRIPTION
**Pull request by guimachiavelli**
Specify user needs to download movies.json file

### CODE REVIEW

File: learn/getting_started/quick_start.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ To be processed, all documents must share one common <clientGlossary word="field
 
 There are [several ways to let MeiliSearch know what the primary key](/learn/core_concepts/documents.md#primary-key) is. The easiest one is to have an <clientGlossary word="attribute" /> that contains the string `id` in a case-insensitive manner.
 
-Below is an example to showcase how to add documents to an index called `movies` using the following test dataset: [movies.json](https://raw.githubusercontent.com/meilisearch/MeiliSearch/master/datasets/movies/movies.json).
+Below is an example to showcase how to add documents to an index called `movies` using the following test dataset: [movies.json](https://raw.githubusercontent.com/meilisearch/MeiliSearch/master/datasets/movies/movies.json). Right-click the movies.json file link and move the downloaded file to your working directory.
```
Comment (dichotommy) [inline learn/getting_started/quick_start.md:62]:
```suggestion
Below is an example to showcase how to add documents to an index called `movies`. To follow along, first right click this link and download the file: [movies.json](https://raw.githubusercontent.com/meilisearch/MeiliSearch/master/datasets/movies/movies.json). Then, move the downloaded file to your working directory.
```

**Review by dichotommy — CHANGES_REQUESTED** [review]
Looks good, just a small wording change!

File: learn/getting_started/quick_start.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ To be processed, all documents must share one common <clientGlossary word="field
 
 There are [several ways to let MeiliSearch know what the primary key](/learn/core_concepts/documents.md#primary-key) is. The easiest one is to have an <clientGlossary word="attribute" /> that contains the string `id` in a case-insensitive manner.
 
-Below is an example to showcase how to add documents to an index called `movies` using the following test dataset: [movies.json](https://raw.githubusercontent.com/meilisearch/MeiliSearch/master/datasets/movies/movies.json).
+Below is an example to showcase how to add documents to an index called `movies`. To follow along, first right click this link and download the file: [movies.json](https://raw.githubusercontent.com/meilisearch/MeiliSearch/master/datasets/movies/movies.json). Then, move the downloaded file to your working directory.
```
Comment (guimachiavelli) [inline learn/getting_started/quick_start.md:62]:
```suggestion
Below is an example to showcase how to add documents to an index called `movies`. To follow along, first right-click this link and download the file: [movies.json](https://raw.githubusercontent.com/meilisearch/MeiliSearch/master/datasets/movies/movies.json). Then, move the downloaded file to your working directory.
```

**Review by dichotommy — APPROVED** [review]
Looks good to me! Since this PR has a few commits (including one unnecessary merge commit), I would recommend using `Rebase & Merge` (should get rid of that merge commit).

**Pull request merged**